views/hearing-tab.html

```diff
@@ -0,0 +1,111 @@
+{% extends "layout.html" %}
+{% from "inset-text/macro.njk" import govukInsetText %}
+{% from "components/navigation-tabs.html" import navigationTabs with context%}
+{% set pageHeader = i18n.taskList.header %}
+{% block corContent %}
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-xl">{{ i18n.taskList.header }}</h1>
+        {% include "includes/appellant-details.html" %}
+    </div>
+</div>
+
+{% include "includes/divider.html" %}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <div class="task-list">
+            {{ navigationTabs('hearing') }}
+            <h2 class="govuk-heading-m --margin-bottom-s">The hearing for your appeal</h2>
+            {% if attending %}
+                {% if not hearingInfo.hearingDateTime %}
+                    <div>You will receive an email when your hearing has been booked.</div>
+                {% endif %}
+                <div class="hearing-details">
+                    <dl class="govuk-summary-list">
+                        <div class="govuk-summary-list__row">
+                            <dt class="govuk-summary-list__key">Date</dt>
+                            <dd class="govuk-summary-list__value">
+                                {{ hearingInfo.hearingDateTime if hearingInfo else 'To be confirmed' }}
+                            </dd>
+                        </div>
+                        <div class="govuk-summary-list__row">
+                            <dt class="govuk-summary-list__key">Time</dt>
+                            <dd class="govuk-summary-list__value">
+                                {{ hearingInfo.hearingDateTime if hearingInfo else 'To be confirmed' }}
+                            </dd>
+                        </div>
+                        <div class="govuk-summary-list__row">
+                            <dt class="govuk-summary-list__key">Location</dt>
+                            <dd class="govuk-summary-list__value">
+                                {% if hearingInfo %}
+                                    {{ hearingInfo.venueName }} <br />
+                                    {{ hearingInfo.addressLine1 }} <br />
+                                    {{ hearingInfo.addressLine2 }} <br />
+                                    {{ hearingInfo.addressLine3 }} <br />
+                                    {{ hearingInfo.postcode }} <br />
+                                    <a href="{{ hearingInfo.googleMapUrl }}">Maps and directions</a>
```

Comment:
And this should it not come from en.json. If so there are a load of places below that should too not going to add a comment to each line.